python/GangaDirac/Dirac.ini

```diff
@@ -4,6 +4,12 @@ RUNTIME_PATH = GangaDirac
 
 ReleaseNotes = False
 
+[defaults_GridCommand]
+info = /cvmfs/ganga.cern.ch/dirac_ui/scripts/dirac-proxy-info
+init = /cvmfs/ganga.cern.ch/dirac_ui/scripts/dirac-proxy-init -g gridpp_user -M
+[DIRAC]
+DiracEnvFile = GangaDirac/cvmfs_DIRAC_env
```

Comment:
This should point at the matching Env file on CVMFS as that will be kept up-to-date with the Dirac client there